integrations/pom.xml

```
@@ -12,7 +12,7 @@
   <name>Configuration as Code Integration tests</name>
 
   <properties>
-    <jenkins.version>2.121.1</jenkins.version> <!-- For artifact-manager-S3. Remove when parent base is updated. -->
+    <jenkins.version>2.161</jenkins.version> <!-- For artifact-manager-S3. Remove when parent base is updated. -->
```

comment is invalid now?
```suggestion
    <jenkins.version>2.161</jenkins.version>
```